provider/data_source_database_role_list_test.go

```diff
@@ -1,66 +0,0 @@
-package provider
```

Comment:
Have to remove tests for the role to avoid too many changes in a PR. Need to update the OpenAPI for role and migrate the provider to the updated role API in the following PRs